_docs/setup/kubernetes/quick-start.md

```diff
@@ -219,7 +219,14 @@ You can now deploy your own application or one of the sample applications provid
 installation like [BookInfo]({{home}}/docs/guides/bookinfo.html).
 Note: the application must use HTTP/1.1 or HTTP/2.0 protocol for all its HTTP traffic because HTTP/1.0 is not supported.
 
-Unless you installed the Istio-Initializer as shown above, when deploying the application, you must
+If you have the Istio-Initializer as shown above installed, then you can deploy the application directly
```

Comment:
nit: remove "then" before you can deploy...

Reply:
Done